udata/core/dataset/models.py

```diff
@@ -217,6 +217,27 @@ def check_availability(self):
         '''
         return self.extras.get('check:available', 'unknown')
 
+    def is_need_check(self):
```

Comment:
I think the `is_` prefix is weird and can be removed (I think this is a convention used on some project but as not in use here I think it's more readable without)

Reply:
👍